test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -124,25 +136,17 @@ class thread_context {
     /*
      * Generic update function, takes a collection_id and key, will generate the value.
      *
-     * Returns true if it successfully updates the key, false if it receives rollback from the API.
+     * Return true if a rollback is required.
```

Comment:
I'm not gonna hold up the PR for this comment, but will fix any usages that I modified in this PR given the outcome of the discussion.

Reply:
I discussed with Jeremy and I it is up to us really. I personally think we should have some consistency and it can be addressed in a different ticket. I prefer `ReturnS`.

Reply:
I think returns is better too actually I'll make it as such.